src/main/java/mate/academy/controllers/orders/GetUserOrdersController.java

```diff
@@ -6,21 +6,23 @@
 import javax.servlet.http.HttpServlet;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
+import javax.servlet.http.HttpSession;
 import mate.academy.lb.Injector;
 import mate.academy.model.Order;
 import mate.academy.service.OrderService;
 
 public class GetUserOrdersController extends HttpServlet {
-    private static final Long USER_ID = 1L;
-
+    private static final String USER_ID = "user_id";
     private static final Injector injector = Injector.getInstance("mate.academy");
     private final OrderService orderService =
             (OrderService) injector.getInstance(OrderService.class);
 
     @Override
     protected void doGet(HttpServletRequest req, HttpServletResponse resp)
             throws ServletException, IOException {
-        List<Order> orderList = orderService.getUserOrders(USER_ID);
+        HttpSession session = req.getSession();
+        Long userId = (Long) session.getAttribute(USER_ID);
```

Comment:
```suggestion
        Long userId = (Long) req.getSession().getAttribute(USER_ID);
```
We use session only once here so maybe we don't need variable for it.

Reply:
agree